cardigan/stories/components/Checkbox.js

```diff
@@ -0,0 +1,11 @@
+import { storiesOf } from '@storybook/react';
+import CheckboxWithLabel from '../../../common/views/components/Checkbox/Checkbox';
+import Readme from '../../../common/views/components/Checkbox/README.md';
+
+const stories = storiesOf('Components', module);
+
+const CheckboxExample = () => <CheckboxWithLabel id="yo" text="Manuscripts" />;
+
+stories.add('Checkbox', CheckboxExample, {
+  readme: { sidebar: Readme },
+});
```

Comment:
Should we have the `CheckboxWithLabel` story in stead, as that's what we have a usecase for?

Reply:
Do you mean rename this story? (it _is_ using the default export – `CheckboxWithLabel`).